exonum-java-binding/core/rust/src/runtime/cmd.rs

```diff
@@ -171,10 +281,67 @@ impl EjbCommand for Run {
     }
 }
 
+/// There are four possible combinations of parameters:
+/// 1. Neither `<db_path>`, nor `<node_config>` are provided. `<tmp>` directory is used for database
+///   files and <tmp>/config for node configuration files.
```

Comment:
That is not complete: _separate_ temporary directories are used for a database and config files.